fiftyone/core/models.py

```diff
@@ -116,6 +119,25 @@ def apply_model(
             **kwargs,
         )
 
+    try:
+        if isinstance(model, ultralytics.YOLO):
+            model = fouu._convert_yolo_model(model)
+            return apply_model(
```

Comment:
Lines 125-137 are unnecessary, right?

Reply:
Ah yep, you're right! That's nice indeed.